README.md

```diff
@@ -2,3 +2,14 @@
 
 # sermon-helper-py
 Helper files for managing sermon files
+
+
+
+# Usage: 
+(BEFORE USING, MAKE SURE TO INSTALL EYED3 PACKAGE IN PYTHON WITH PIP)- Ask Mark how to do this.
+
+Copy Audacity edited mp3 file to same folder as tagmp3.py.
+Copy sermon-example.ini to sermon.ini.
+Edit values in sermon.ini to values to match the desired tags for the sermon this week. 
+Run script in terminal, and your mp3 will be copied and tagged with the correct tags. 
```

Comment:
Could you add a reference on how to run the script (`python tagmp3.py` or `./tagmp3.py`)?